controllers/perconaservermysql_controller.go

```diff
@@ -1077,58 +1066,13 @@ func (r *PerconaServerMySQLReconciler) reconcileGroupReplication(ctx context.Con
 		return errors.Wrap(err, "get operator password")
 	}
 
-	db, err := replicator.NewReplicator(apiv1alpha1.UserOperator, operatorPass, mysql.FQDN(cr, 0), mysql.DefaultAdminPort)
-	if err != nil {
-		return errors.Wrapf(err, "connect to %s", firstPod.Name)
-	}
-	defer db.Close()
-
-	asyncReplicationStatus, _, err := db.ReplicationStatus()
-	if err != nil {
-		return errors.Wrapf(err, "get async replication status of %s", firstPod.Name)
-	}
-	if asyncReplicationStatus == replicator.ReplicationStatusActive {
-		l.Info("Replication threads are running. Stopping them before starting group replication", "pod", firstPod.Name)
-		if err := db.StopReplication(); err != nil {
-			return err
-		}
-	}
-
-	firstPodFQDN := fmt.Sprintf("%s.%s.%s", firstPod.Name, mysql.ServiceName(cr), cr.Namespace)
-	firstPodUri := fmt.Sprintf("%s:%s@%s", apiv1alpha1.UserOperator, operatorPass, firstPodFQDN)
-
-	mysh := mysqlsh.New(k8sexec.New(), firstPodUri)
+	uri := fmt.Sprintf("%s:%s@%s", apiv1alpha1.UserOperator, operatorPass, mysql.ServiceName(cr))
+	mysh := mysqlsh.New(k8sexec.New(), uri)
 
 	clusterExists, err := mysh.DoesClusterExist(ctx, cr.InnoDBClusterName())
 	if err != nil {
-		if errors.Is(err, mysqlsh.ErrMetadataExistsButGRNotActive) {
-			l.Info("Rebooting cluster from complete outage")
-			if err := mysh.RebootClusterFromCompleteOutage(ctx, cr.InnoDBClusterName(), []string{firstPodFQDN}); err != nil {
-				return err
-			}
-			l.Info("Successfully rebooted cluster")
-			return nil
-		}
 		return errors.Wrapf(err, "check if InnoDB Cluster %s exists", cr.InnoDBClusterName())
 	}
-	state := innodbcluster.MemberStateOffline
-	// it's not possible to run Cluster.status() on a standalone MySQL node
-	if clusterExists {
-		instance := fmt.Sprintf("%s:%d", firstPodFQDN, mysql.DefaultPort)
-		state, err = mysh.MemberState(ctx, cr.InnoDBClusterName(), instance)
-		if err != nil {
-			return errors.Wrapf(err, "get member state of %s", firstPod.Name)
-		}
-	}
-
-	l.V(1).Info(fmt.Sprintf("First pod state: %s", state))
-	if state == innodbcluster.MemberStateOffline {
-		l.Info("Configuring first instace", "instace", firstPodFQDN)
-		if err := mysh.ConfigureInstance(ctx, firstPodUri); err != nil {
-			return err
-		}
-		l.Info("Configured first instance", "instance", firstPodFQDN)
-	}
 
 	if !clusterExists {
```

Comment:
Is this valid now that we have bootstrap? Should pods handle this state themselves?

If it is valid, do we need to check `mysqlsh.ErrMetadataExistsButGRNotActive` so we can do `mysh.RebootClusterFromCompleteOutage`. Can it happen?

Reply:
recovering from full cluster crash is performed by pod-0 in the bootstrap. no need to do it from operator any more